src/components/BShuttle/BShuttle.vue

```diff
@@ -462,7 +465,7 @@ export default {
 
   li.b-shuttle-options {
     padding: .4rem 0rem .4rem .75rem;
-    font-size: 14px;
+    font-size: 85.7%;
```

Comment:
@eleanormal this ok with you?

Reply:
should this be `percent` or `rem`s?

Reply:
We talked as a team about using `rem` so that's what this should be. But if the body size is set at 14px then this won't need a font-size